dynawo/sources/API/PAR/PARParametersSet.h

```diff
@@ -191,6 +192,14 @@ class ParametersSet : public boost::enable_shared_from_this<ParametersSet> {
    */
   boost::shared_ptr<ParametersSet> addParameter(const boost::shared_ptr<Parameter>& param);
 
+  /**
+   * @brief Add a new macroParSet in macroParSets_ map if it does not already exists
```

Comment:
Please avoid refering to internal structures in description: Add a new macroParSet to the parameter set

Reply:
Please also mentions that this throws if the id is already used

Reply:
ok, thanks